includes/api/legacy/v2/class-wc-api-products.php

```diff
@@ -378,9 +378,8 @@ public function edit_product( $id, $data ) {
 				if ( isset( $data['variations'] ) && is_array( $data['variations'] ) ) {
 					$this->save_variations( $product, $data );
 				} else {
-					// Just sync variations
-					WC_Product_Variable::sync( $id );
-					WC_Product_Variable::sync_stock_status( $id );
+					// Just sync variations.
+					WC_Product_Variable::sync( $product );
```

Comment:
Just flagging this since it might need changed along with https://github.com/woocommerce/woocommerce/pull/12329#pullrequestreview-8250578